website/project/model.py

```diff
@@ -1964,7 +1964,7 @@ def get_recent_logs(self, n=10):
 
         :param int n: Number of logs to retrieve
         """
-        return list(reversed(self.logs)[:n])
+        return list(reversed(self.logs))[:n]
```

Comment:
are we sure we want to `list(...)` the entirety of `self.logs`? seems like it could be a big number...